inference/core/active_learning/README.md

```diff
@@ -0,0 +1,269 @@
+# Active Learning
+
+This feature is designed to facilitate the systematic collection of data from actual production distributions with 
+minimal user-side overhead, leveraging the hosting capabilities of models and datasets on the Roboflow platform.
+
+The core concept is straightforward:
```

Comment:
Remove the word "straightforward".

See https://jamesg.blog/2023/04/17/technical-communication/ for more information.